graphistry/feature_utils.py

```diff
@@ -114,6 +143,35 @@ def assert_imported():
         raise import_min_exn
 
 
+def assert_cuml_cucat():
+    has_cuml_dependancy_, import_cuml_exn, cudf = lazy_import_has_dependancy_cuda()
```

Comment:
i think this is just `has_dependency, import_exn, cudf = ..`

but see note above, the method signature is confusing, which is making it tricky to review (& maintain) this code